st2api/st2api/controllers/v1/webhooks.py

```diff
@@ -51,11 +51,12 @@ def __init__(self, *args, **kwargs):
         self._trigger_types = WEBHOOK_TRIGGER_TYPES.keys()
 
         self._trigger_dispatcher = TriggerDispatcher(LOG)
+        queue_suffix = self.__class__.__name__ + ':' + str(uuid.uuid4())
```

Comment:
The name might be too long, in get_queue function we actually truncate the uuid. Probably a good idea to do that here as well.

```python
    def _get_queue(queue_suffix, exclusive):
        if not queue_suffix:
            # pick last 10 digits of uuid. Arbitrary but unique enough for the TriggerWatcher.
            u_hex = uuid.uuid4().hex
            queue_suffix = uuid.uuid4().hex[len(u_hex) - 10:]
        queue_name = 'st2.trigger.watch.%s' % queue_suffix
```